tools/sotn_str/jp.py

```diff
@@ -119,6 +119,11 @@ def dakuten(chr, prev):
 for index, value in enumerate(table):
     utf8_to_index[value] = index
 
+# ASCII characters not mapped above, but useful for encoding strings in C files.
+ascii_supplement = "*;<=>@"
+for c in ascii_supplement:
+    utf8_to_index[c] = ord(c) - 0x20
+
```

Comment:
Entries in `g_MenuStr` have several adjacent ASCII characters, but in the mapping table above some of those have non-ascii characters in their positions. This allows these ASCII characters to be used as well when it makes sense.

Reply:
I'm not sure I'm understanding why this is needed, the mapping is like this
![image](https://github.com/Xeeynamo/sotn-decomp/assets/122322823/e8ae73dc-f4ad-4387-ba25-305556f5c2e2)
These characters are not in the table?

Reply:
It's not relevant after removing the `menu.c` change.